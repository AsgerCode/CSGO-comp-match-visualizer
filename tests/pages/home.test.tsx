import { render, screen } from '@testing-library/react';
import Home from '../../src/pages/index';

jest.mock('next/router', () => require('next-router-mock'));

describe('Home', () => {
  it('renders the file upload input', () => {
    const { container } = render(<Home />);
    const input = screen.getByRole('fileInput');
    expect(input).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});