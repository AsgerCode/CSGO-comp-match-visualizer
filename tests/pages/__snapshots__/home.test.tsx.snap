// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the file upload input 1`] = `
<div>
  <div
    class="flex flex-row items-center justify-center h-screen"
  >
    <div
      class="mr-6"
    >
      Upload a CS:GO match data file to have it analyzed.
    </div>
    <input
      accept=".txt"
      id="fileInput"
      role="fileInput"
      type="file"
    />
  </div>
</div>
`;
